sdk/lib/drivers/wdf/shared/core/fxdevice.cpp

```diff
@@ -1546,6 +1547,7 @@ DispatchWorker(
 
 _Must_inspect_result_
 NTSTATUS
+STDCALL
```

Comment:
Add `#ifdef __REACTOS__` to all MS WDF changes?

Reply:
Not done previously. I'll let this decision to @Extravert-ir